inst/templates/Makevars.in

```diff
@@ -0,0 +1,44 @@
+TARGET_DIR = ./rust/target
+LIBDIR = $(TARGET_DIR)/release
+STATLIB = $(LIBDIR)/lib{{{lib_name}}}.a
```

Comment:
Unrelated to things, but I think this could be named `RUSTLIB` without any issue..

Reply:
Do you think doing that could help the possibility of including Rcpp + extendr in the same package?

Reply:
Let's have a meeting about this at some point. I think I know what needs to be investigated to do this.